filebeat/tests/system/test_modules.py

```diff
@@ -140,7 +140,7 @@ def run_on_file(self, module, fileset, test_file, cfgfile):
         output.write(" ".join(cmd) + "\n")
 
         local_env = os.environ.copy()
-        local_env["TZ"] = 'Etc/UTC'
+        local_env["TZ"] = 'Etc/GMT+2'
```

Comment:
it would be nice to add a comment here explaining why this timezone

Reply:
Agree, comment added.